src/index.html

```diff
@@ -1,18 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Leaderboard App</title>
 </head>
+
 <body>
     <header>
         <nav>
             <ul id="navigation">
                 <li class="navItem">Leaderboard</li>
                 <li class="navItem">New</li>
-                <li class="navItem mobile"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Pro 6.1.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. --><path d="M0 96C0 78.33 14.33 64 32 64H416C433.7 64 448 78.33 448 96C448 113.7 433.7 128 416 128H32C14.33 128 0 113.7 0 96zM0 256C0 238.3 14.33 224 32 224H416C433.7 224 448 238.3 448 256C448 273.7 433.7 288 416 288H32C14.33 288 0 273.7 0 256zM416 448H32C14.33 448 0 433.7 0 416C0 398.3 14.33 384 32 384H416C433.7 384 448 398.3 448 416C448 433.7 433.7 448 416 448z"/></svg></li>
+                <li class="navItem mobile"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
+                        <!--! Font Awesome Pro 6.1.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. -->
+                        <path
+                            d="M0 96C0 78.33 14.33 64 32 64H416C433.7 64 448 78.33 448 96C448 113.7 433.7 128 416 128H32C14.33 128 0 113.7 0 96zM0 256C0 238.3 14.33 224 32 224H416C433.7 224 448 238.3 448 256C448 273.7 433.7 288 416 288H32C14.33 288 0 273.7 0 256zM416 448H32C14.33 448 0 433.7 0 416C0 398.3 14.33 384 32 384H416C433.7 384 448 398.3 448 416C448 433.7 433.7 448 416 448z" />
+                    </svg></li>
```

Comment:
- [ ] The leaderboard app does not have any navigation, so that renders this whole `nav` section needless because it's not being used. Kindly remove it.
- [ ] Let's try to stick as closely as possible to the provided wireframe. It is important for devs to be able to follow design specifications as that will help you in the real world to ensure client satisfaction.
- [ ] Please note that your form input elements and the button should be placed vertically and not horizontally like you've done 🙏 
- [ ] Please modify your CSS to get rid of horizontal scrolling as it is bad for the user experience.
Kindly reference the annotated design image below and make the changes which I have marked in green 👇 
### Wireframe
![image](https://user-images.githubusercontent.com/46839548/187032493-3a7b18b6-9f64-49d8-ac0d-a98d245df237.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/187032744-8b3c6794-54a3-4e6e-9517-f36fc2b27c50.png)